src/tracer.js

```diff
@@ -207,3 +219,19 @@ module.exports.sendTraceSync = function sendTraceSync() {
 
     return sendCurrentTrace(traceObject => postTrace(traceObject));
 };
+
+/**
+ * Add a custom label to the runner of the current trace.
+ * @param {string} key key for the added label
+ * @param {string} value value for the added label
+ */
+module.exports.label = function addLabel(key, value) {
+    if (typeof key !== 'string' || typeof value !== 'string') {
+        return;
+    }
+
+    if (key.length + value.length + currentLabelsLen <= consts.MAX_LABEL_SIZE) {
+        eventInterface.addLabelToMetadata(currRunner, key, value);
+        currentLabelsLen += key.length + value.length;
```

Comment:
why do you need currentLabelsLen?

Reply:
to avoid dynamically calculating the size of the map each time
is there any nice JS way to do it? (couldn't find something)

Reply:
we talked about it, fix it <3